config/nimdoc.cfg

```diff
@@ -132,7 +132,10 @@ doc.body_toc_group = """
         <a href="$theindexhref">Index</a>
       </li>
       <li>
-        <a href="compiler/$theindexhref">Compiler docs</a>
+        <a href="compiler/$theindexhref">compiler docs</a>
```

Comment:
Why the change of case here?

Reply:
it's for consistency with `import compiler/foo` (instead of `import Compiler/foo`), ditto with `import fusion/bar` instead of `import Fusion/bar`

I don't care much either way here.

still though, I would've preferred to get  nim-lang/fusion#26  merged first as indicated in top post to make sure the link is valid